### PR TITLE
update ipwl to ga for update and create and fix linter warnings

### PR DESCRIPTION
**Pull request by gtracer**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/_help.py [hunk 1]
```diff
@@ -358,12 +368,9 @@
           type: int
           short-summary: The maximum number of pods deployable to a node.
           long-summary: If not specified, defaults to 110, or 30 for advanced networking configurations.
-        - name: --node-zones
```
Comment (robbiezhang) [inline src/aks-preview/azext_aks_preview/_help.py]: --node-zones is removed from the azure-cli. please also remove it here.

**Comment by robbiezhang** [comment]
Just 1 comment about removing the --node-zones. otherwise, LGTM

File: src/aks-preview/azext_aks_preview/_help.py (hunk 1 above)
Comment (adilliadil) [reply src/aks-preview/azext_aks_preview/_help.py]: We agreed with @jluk that we would do the removal in two steps.
1. add --zones -z as possible parameters also indicate in the help that --node-zones would be deprecated. (PR already merged)
2. remove --node-zones (will send out another PR)
this is for the purpose that the customers are aware of the changes we make to extension and have time to make the adjustments.

**Comment by ruchikaguptaa** [comment]
LGTM.

**Pull request merged**